yarn-project/noir-contracts/scripts/install_noir_backend.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Script to install the current barretenberg backend
+
+set -eu
+
+BACKEND_NAME="acvm-backend-barretenberg"
+
+# If the backend is not installed, then install it
+if [ -z $(nargo backend ls | grep $BACKEND_NAME) ]; then
+    echo "Installing $BACKEND_NAME"
+    nargo backend install $BACKEND_NAME https://github.com/AztecProtocol/barretenberg/releases/download/barretenberg-v0.5.1/barretenberg-x86_64-linux-gnu.tar.gz
```

Comment:
If there was no parallel compile, could we revert this PR?

Asking because:

- Theres a PR to compile contracts in parallel by Tom 
- This is likely to be a footgun in the future

Reply:
Not a blocker on this PR

Reply:
Yep i think so